CHANGELOG.md

```diff
@@ -7,11 +7,12 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Add an `isAdmin` property to `RLMSyncUser` indicating whether a user is a
+  Realm Object Server administrator.
 
 ### Bugfixes
 
-* None.
+* Fix an issue where the project would fail to compile with clang 4.0.
```

Comment:
This isn't relevant to Cocoa at the moment since no Xcode version exists that uses Clang 4.0.